CHANGELOG.md

```diff
@@ -21,6 +21,9 @@ different versioning scheme, following the Haskell community's
 
 ### `gbc` and Bond compiler library ###
 
+* C++ codegen generates field reflection, that uses an explicit type 
+  derrived from reflection::FieldTemplate. This hides FieldTemplate 
+  details and, as result, shortens symbol names.
```

Comment:
Minor: Change to "C++ codegen hides FieldTemplate details, shortening symbol names."